<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>pat</class>
 <widget class="QStackedWidget" name="pat">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>763</width>
    <height>597</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>StackedWidget</string>
  </property>
  <property name="currentIndex">
   <number>1</number>
  </property>
  <widget class="QWidget" name="page_pat_home">
   <widget class="QPushButton" name="page_pat_home_logout">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>10</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Logout</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="page_pat_home_textEdit_search">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>70</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Search box&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QTreeWidget" name="page_pat_home_treeWidget_treatment_list">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>100</y>
      <width>461</width>
      <height>451</height>
     </rect>
    </property>
    <property name="sortingEnabled">
     <bool>true</bool>
    </property>
    <column>
     <property name="text">
      <string>Patient ID</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Date ordered</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Order ID</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Status</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="page_pat_home_button_my_profile">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>100</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>My profile </string>
    </property>
   </widget>
   <widget class="QPushButton" name="page_pat_home_button_proceed">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>550</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Proceed</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="page_pat_errand">
   <widget class="QTreeWidget" name="page_pat_errand_treeWidget_errand_list">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>80</y>
      <width>331</width>
      <height>381</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>331</width>
      <height>0</height>
     </size>
    </property>
    <property name="sortingEnabled">
     <bool>true</bool>
    </property>
    <column>
     <property name="text">
      <string>Task</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Date</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Scan</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Status</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Clinic</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="page_pat_errand_button_back">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>520</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Back</string>
    </property>
   </widget>
   <widget class="QPushButton" name="page_pat_errand_button_view_status">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>460</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>View Status</string>
    </property>
   </widget>
   <widget class="QPushButton" name="page_pat_errand_button_share">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>460</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Share</string>
    </property>
   </widget>
   <widget class="QPushButton" name="page_pat_errand_button_download">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>460</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Download</string>
    </property>
   </widget>
   <widget class="QPushButton" name="page_pat_errand_button_view">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>460</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>View</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="page_pat_errand_report">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>371</width>
      <height>401</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Display radiology report here</string>
    </property>
   </widget>
   <widget class="QPushButton" name="page_pat_errand_logout">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>10</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Logout</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="page_pat_errand_report_textEdit_search">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>50</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="plainText">
     <string>Search</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="page_pat_my_profile">
   <widget class="QPushButton" name="page_pat_my_profile_button_back">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>520</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Back</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="page_pat_my_profile_textBrowser">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>40</y>
      <width>401</width>
      <height>491</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="page_pat_image_status">
   <widget class="QTextBrowser" name="page_pat_image_status_textBrowser_detailed_information">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>380</y>
      <width>431</width>
      <height>191</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="page_pat_image_status_button_back">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>520</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Back</string>
    </property>
   </widget>
   <widget class="QPushButton" name="page_pat_image_status_button_request_notification">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>460</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Request notifications</string>
    </property>
   </widget>
   <widget class="QPushButton" name="page_pat_image_status_button_logout">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>10</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Logout</string>
    </property>
   </widget>
   <widget class="QOpenGLWidget" name="page_pat_image_status_journey_image">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>30</y>
      <width>531</width>
      <height>331</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="page_pat_view_scan">
   <widget class="QPushButton" name="page_pat_view_scan_button_previous_note">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>380</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Previous Note</string>
    </property>
   </widget>
   <widget class="QPushButton" name="page_pat_view_scan_button_previous_slice">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>410</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Previous Slice</string>
    </property>
   </widget>
   <widget class="QPushButton" name="page_pat_view_scan_button_play_pause">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>380</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Play/Paus</string>
    </property>
   </widget>
   <widget class="QPushButton" name="page_pat_view_scan_button_next_note">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>380</y>
      <width>75</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Next Note</string>
    </property>
   </widget>
   <widget class="QPushButton" name="page_pat_view_scan_button_next_slice">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>410</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Next Slice</string>
    </property>
   </widget>
   <widget class="QPushButton" name="page_pat_view_scan_button_right">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>300</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Right</string>
    </property>
   </widget>
   <widget class="QPushButton" name="page_pat_view_scan_button_left">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>270</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Left</string>
    </property>
   </widget>
   <widget class="QPushButton" name="page_pat_view_scan_button_up">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>270</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Up</string>
    </property>
   </widget>
   <widget class="QPushButton" name="page_pat_view_scan_button_down">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>300</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Down</string>
    </property>
   </widget>
   <widget class="QPushButton" name="page_pat_view_scan_button_zoom_in">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>270</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Zoom in</string>
    </property>
   </widget>
   <widget class="QPushButton" name="page_pat_view_scan_button_zoom_out">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>300</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Zoom out</string>
    </property>
   </widget>
   <widget class="QPushButton" name="page_pat_view_scan_button_3d_fullscreen">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>30</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>full screen</string>
    </property>
   </widget>
   <widget class="QPushButton" name="page_pat_view_scan_button_3d_bone_view">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>30</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Bone View</string>
    </property>
   </widget>
   <widget class="QPushButton" name="page_pat_view_scan_button_3d_tissue_view">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>30</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Tissue View</string>
    </property>
   </widget>
   <widget class="QPushButton" name="page_pat_view_scan_button_logout">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>10</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Logout</string>
    </property>
   </widget>
   <widget class="QPushButton" name="page_pat_view_scan_button_back">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>540</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Back</string>
    </property>
   </widget>
   <widget class="QFrame" name="page_pat_view_scan_2d_view_frame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>341</width>
      <height>311</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QWidget" name="page_pat_view_scan_2d_view" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>341</width>
       <height>311</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="page_pat_view_scan_3d_view_frame">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>60</y>
      <width>301</width>
      <height>201</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QWidget" name="page_pat_view_scan_3d_view" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>301</width>
       <height>201</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="page_pat_view_scan_rad_annotations_frame">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>330</y>
      <width>301</width>
      <height>231</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QWidget" name="page_pat_view_scan_rad_annotations" native="true">
     <property name="geometry">
      <rect>
       <x>-1</x>
       <y>-1</y>
       <width>301</width>
       <height>231</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
